nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -1,45 +1,21 @@
 ---
 title: Configuration
+component: ASP
 summary: Configuring Azure Storage as persistence
 tags:
  - Azure
  - Cloud
  - Persistence
+reviewed: 2016-11-07
 ---
 
 In NServiceBus.Azure the behavior of the `AzureStoragePersister` can be controlled by working with the appropriate configuration section(s) in the `app.config` or by using the code via the [`IProvideConfiguration` adapter](/nservicebus/hosting/custom-configuration-providers.md). Both approaches to configuring the persister can access the same configuration options.
```

Comment:
The page is already linked, it's https://docs.particular.net/nservicebus/hosting/custom-configuration-providers which describes the order or resolution.